sdk/cognitivelanguage/azure-ai-language-text/swagger/README.md

```diff
@@ -0,0 +1,99 @@
+# Azure Text Analytics Authoring Client for Python
```

Comment:
Take a look at the CLU readme for how to rename some of the operations and remove the operation group: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md?plain=1#L291

The generator will create an operation group if the operationId has a prefix like "TextAnalysisAuthoring" in "TextAnalysisAuthoring_ListProjects". If we remove that prefix when we rename operations, then the generator will generate all the operations directly on the client. So `client.text_analysis_authoring.list_projects` becomes simply `client.list_projects`.